scripts/export_gen3_hsc.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import lsst.daf.butler as dafButler
+from lsst.daf.butler import CollectionType
+import re
+
+butler = dafButler.Butler('hsc/butler.yaml')
+
+collection = 'HSC/testdata'
```

Comment:
For consistency it might help to do something like:
```
>>> from lsst.obs.subaru import HyperSuprimeCam
>>> HyperSuprimeCam.makeCollectionName("testdata")
'HSC/testdata'
```
You can also do that in the config file above.